update.sh

```diff
@@ -15,5 +15,6 @@ openstack overcloud update prepare --stack overcloud \
 		-e enable-tls.yaml \
 		-e tls-endpoints-public-dns.yaml \
 		-e cloudname.yaml \
-		-e esi-custom.yaml
-
+		-e esi-custom.yaml \
+		-e keystone_extraconfig.yaml \
+        -e sso_vars.yaml
```

Comment:
Should we update deploy.sh as well, in case we ever do a fresh install? (If so, may as well fix the spacing here to be consistent)

Reply:
Yes. I've updated deploy.sh as well.